src/plugins.js

```diff
@@ -32,15 +32,20 @@ var _ = Mavo.Plugins = {
 				}
 
 				// Load plugin
-				var filename = `mavo-${plugin.id}.js`;
+				let filename = `mavo-${plugin.id}.js`;
+				let url;
 
-				if (plugin.repo) {
+				if (plugin.url) {
+					// The plugin author pointed out where the plugin is hosted
+					url = plugin.url;
```

Comment:
If we come to the conclusion that `plugin.url` is a base URL for all the plugin files and is not include `filename`, I'll replace this line with:
```js
url = `${plugin.url}/${filename}`
```

Reply:
Was there any further discussion on that?

Reply:
Yep, it's here: https://github.com/mavoweb/plugins/pull/85#discussion_r569444932